File: ProjetoPOO/src/Pessoa.java
public class Pessoa {
    protected String nome;
    protected int idade;
    protected String sexo;
    protected int experiencia;

    protected void ganharExp(){
        this.experiencia++;
    }

    public Pessoa(String nome, int idade, String sexo, int experiencia) {
        this.nome = nome;
        this.idade = idade;
        this.sexo = sexo;
        this.experiencia = experiencia;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    public String getSexo() {
        return sexo;
    }

    public void setSexo(String sexo) {
        this.sexo = sexo;
    }

    public boolean isExperiencia() {
        return experiencia;
    }

    public void setExperiencia(boolean experiencia) {
        this.experiencia = experiencia;
    }
}

File: ProjetoGafanhotoKotlin/src/main/kotlin/Pessoa.kt
open class Pessoa(
    protected var nome: String,
    protected var idade: Int,
    protected var sexo: String,
    protected var experiencia: Float
) {
    protected fun ganharExp() {
        experiencia++
    }

    override fun toString(): String {
        return "Pessoa [nome=$nome, idade=$idade, sexo=$sexo, experiencia=$experiencia]"
    }
}
